packages/react-textarea/.storybook/main.js

```diff
@@ -0,0 +1,14 @@
+const rootMain = require('../../../.storybook/main');
```

Comment:
could you please bootstrap the auto generated stuff in the package in a separate PR to remove as much useless changes as possible for this review ?